Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM docker-upgrade.artifactory.build.upgrade.com/container-base:20180917-9
+LABEL maintainer=devops@upgrade.com
+
+USER root
```

Comment:
please don't! :)

Reply:
in order to install python3.6, I needed to add epel7 repo. and it needs root privilege 
`You lack permissions to write to system configuration.  /etc/yum.repos.d/amzn2-extras.repo`

Reply:
That's fine but you need to go back to a non root user after those operations.

Reply:
sure. understood